src/modules/rlm_python/rlm_python.c

```diff
@@ -205,6 +206,58 @@ static void mod_error(void)
 	Py_XDECREF(pTraceback);
 }
 
+static int dlopen_linked_libpython_cb(struct dl_phdr_info *info,
+					UNUSED size_t size, void *data)
+{
+	const char *pattern = "/libpython"
+				STRINGIFY(PY_MAJOR_VERSION) "."
+				STRINGIFY(PY_MINOR_VERSION) ".so";
```

Comment:
Hi Arran,

I'm not sure what you mean, but I'll make a guess :)
This is matching the beginning of the actual linked filename.
E.g. pattern might be:

    /libpython2.7.so

And the matched path might be:

    /usr/lib/x86_64-linux-gnu/libpython2.7.so.1.0

This is to make it match the exact python version we were building with (not really necessary, as it's linked already) and to prevent matching against something like libpython-blah.so (which, e.g. can be linked by libpython in its own turn), if we would only use "/libpython" pattern, without the version.